internal/api-key/command_delete.go

```diff
@@ -32,7 +32,7 @@ func (c *command) delete(cmd *cobra.Command, args []string) error {
 		return err == nil
 	}
 
-	if err := deletion.ValidateAndConfirmDeletionYesNo(cmd, args, existenceFunc, resource.ApiKey); err != nil {
+	if err := deletion.ValidateAndConfirmDeletion(cmd, args, existenceFunc, resource.ApiKey); err != nil {
```

Comment:
Since the package is called "deletion" already, I'm ok removing the second "deletion"
```suggestion
	if err := deletion.ValidateAndConfirm(cmd, args, existenceFunc, resource.ApiKey); err != nil {
```

Reply:
I think we can leave this one because we also have `ValidateAndConfirmUndeletion`.